kepler/tasks.py

```diff
@@ -13,6 +13,7 @@
 """
 
 from __future__ import absolute_import
+import xml.etree.ElementTree as ET
```

Comment:
We're already using lxml.etree down below, so might as well just use that. Instead of using `ET.fromstring` use `etree.fromstring`. A few more notes about lxml further down.